### PR TITLE
fix(normalize): check header value and enable integer values also

### PR DESCRIPTION
**Pull request by blacksonic**

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -3,6 +3,8 @@
 var crypto = require('crypto');
 var url = require('url');
 var formatDate = require('dateformat');
+var isNumber = require('is-number');
```
Comment (gergelyke) [inline lib/utils.js]: why not `isNaN`?

Comment (gergelyke) [inline lib/utils.js]: never mind, got it :)

Comment (blacksonic) [inline lib/utils.js]: isNaN(null)
// false

File: .npmignore [hunk 2]
```diff
@@ -0,0 +1,34 @@
+node_modules
+
+.jshintrc
+.travis.yml
+.jsbeautifyrc
+release.sh
+
+spec
+
+.idea
```
Comment (gergelyke) [inline .npmignore]: it should be kept in a global `.gitignore`

**Pull request merged**